app/src/main/java/org/jellyfin/mobile/media/MediaService.kt

```diff
@@ -160,20 +159,18 @@ class MediaService : MediaBrowserServiceCompat() {
                                 quality = 90
                             })
 
-                            val description: MediaDescriptionCompat =
-                                MediaDescriptionCompat.Builder().apply {
-                                    setMediaId(MediaItemType.Library.toString() + "|" + item.id)
-                                    setTitle(item.name)
-                                    setIconUri(Uri.parse(itemImageUrl))
-                                }.build()
+                            val description: MediaDescriptionCompat = MediaDescriptionCompat.Builder().apply {
```

Comment:
```suggestion
                            val description = MediaDescriptionCompat.Builder().apply {
```

Reply:
Not even part of my changes, lol. Still, will fix that.